Jobtypes/jobQueue.js

```diff
@@ -1,12 +1,23 @@
-export { JobQueue , Job};
+export { JobQueue, Job };
 import fs from "fs";
 
 import { createFolder } from "../utility.js";
-import { addMatrixToQue} from "./matrix_multiplication/jobCreateMatrixMult.js";
+import { addMatrixToQue } from "./matrix_multiplication/jobCreateMatrixMult.js";
 import { addPlusToQue } from "./plus/jobCreatePlus.js";
 
+/**
+ * Constructor for a job object in the job-queue. Note that each job includes two linked list queues,
+ * `subtaskList` and `pendingList` for subtasks.
+ * @constructor
+ * @param {string} jobId - The Id of the job.
+ * @param {string} jobType - The type of the job.
+ * @param {string} jobOwner - The owner of the job.
+ * @param {string} algorithm - The algorithm used for the job formatted as string.
```

Comment:
måske lige tilføje at det er hvis der er noget data der er det samme for alle subtasks